### PR TITLE
Theming Storybook with an Emulsify theme

### PR DESCRIPTION
**Pull request by amazingrando**
## Theming Storybook with an Emulsify theme

This PR styles the Storybook interface to "brand" it as Emulsify. It is still obviously Storybook, but now it looks like Emulsify's implementation of Storybook.

## Ticket
https://app.clickup.com/t/36718269/EMULSIFY-56

## How to review this pull request
- [ ] Run `npm run develop`
- [ ] Confirm the interface looks like the image below.
- [ ] Explore the interface—open controls, click on buttons, etc—and confirm there is not something that looks weird or off. (I think I caught all of the edge cases, but I appreciate your eyes on this.)

CleanShot 2023-05-25 at 10 57 22@2x

### CODE REVIEW

**Review by codechefmarc — CHANGES_REQUESTED** [review]
Most of this looks good, theming looks as expected. A few spots I noted with screenshots below have color contrast issues, however:
1. I received a notice that Storybook needed an update via a notification bubble:
Screenshot 2023-06-21 at 6 53 51 AM
2. Clicking through to where the update notes are (or clicking the three dots in the top next to the Emulsify logo) then clicking about Storybook:
 Screenshot 2023-06-21 at 6 53 58 AM
3. Controls for components have some color contrast for the name of the control, and then the text boxes are red (This was for the Accordion component):
Screenshot 2023-06-21 at 6 55 08 AM
4. Keyboard shortcuts area:
Screenshot 2023-06-21 at 6 59 41 AM

**Comment by mariannuar** [comment]
This ticket is ready for review again. Whoever reviews it, please take it in count that font-sizes values are not being displayed as expected. For example, the cover looks terrible but it's because of that. There's another PR that fixes this. https://github.com/emulsify-ds/emulsify-ui-kit/pull/114

Screenshot 2024-02-13 at 10 30 55 PM

File: .storybook/emulsifyTheme.js [hunk 1]
```diff
@@ -3,10 +3,38 @@
...
+  inputBorder: 'silver',
+  inputTextColor: '#00202E',
+  inputBorderRadius: 4,
+  
   // Branding
   brandTitle: 'Emulsify',
   brandUrl: 'https://emulsify.info',
   brandImage:
-    'https://raw.githubusercontent.com/emulsify-ds/emulsify-design-system/master/images/logo.png',
+    'https://randyoest.com/random/emulsify-logo-sb.svg',
```
Comment (amazingrando) [inline .storybook/emulsifyTheme.js:38]: I have to replace this image! Yikes!

**Review by amazingrando — COMMENTED** [review]
I have a single change to make and this is approved.

**Pull request merged**